src/platforms/windows/windowsnetworkwatcher.cpp

```diff
@@ -148,26 +139,6 @@ void WindowsNetworkWatcher::processWlan(PWLAN_NOTIFICATION_DATA data) {
 }
 
 NetworkWatcherImpl::TransportType WindowsNetworkWatcher::getTransportType() {
-  using namespace winrt::Windows::Networking::Connectivity;
-  ConnectionProfile profile =
-      NetworkInformation::GetInternetConnectionProfile();
-
-  if (profile.IsWlanConnectionProfile()) {
-    return TransportType_WiFi;
-  }
-  if (profile.IsWwanConnectionProfile()) {
-    return TransportType_Cellular;
-  }
-  NetworkAdapter device = profile.NetworkAdapter();
-  if (device == nullptr) {
-    return TransportType_Unknown;
-  }
-
-  switch (device.IanaInterfaceType()) {
-    case 6:
-      return TransportType_Ethernet;
-    case 71:
-      return TransportType_WiFi;
-  }
+  // TODO: Implement this once we update to Qt6.3 (VPN-3511)
```

Comment:
why this change?

Reply:
I am not sure you saw the message I left on the commit explaining the change, so from the commit message:

```
The version of cppwinrt that we use is v2.0.200609.3
and it comes from Windows SDK 10.0.19041.0.

That version of cppwinrt has a bug for suppoting C++20,
which is fixed on version 2.0.200921.6.

It's unclear wheather it is safe for us to update to a Windows SDK
version that includes the correct version of cppwinrt, because that
would require us to be on a Windows 11 SDK. We are unsure whether
building with that will result in binaries that can also be run on
Windows 10.

After speaking to Basti, we decided the least problematic way to address
this is to simply remove the code that relies on cppwinrt. That will be
removed anyways as soon as we more to Qt6.3 or above, because Qt will
provide the APIs we are currently getting from cppwinrt.

That code is only being called to add an extra key to a Glean event
recording, so this doesn't actually affect user facing features on the
app.
```

Let me know if that is still unclear.